### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by LowderPlay**
- [x] Added analytics
- [x] Added biometric unlock
- [ ] ~~Add global announcements~~ (next release)

### CODE REVIEW

**Comment by LowderPlay** [comment]
Release preview is available at https://t.me/TeleOTPAppBot/preview

**Review by LowderPlay — COMMENTED** [review]
LGTM 😎

File: src/main.tsx [hunk 1]
```diff
@@ -48,12 +50,17 @@ const router = createBrowserRouter(
 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 ReactDOM.createRoot(document.getElementById('root')!).render(
   <React.StrictMode>
+    <PlausibleAnalyticsProvider domain="teleotp.pages.dev" apiHost="https://analytics.gesti.tech/">
```
Comment (lenchq) [inline src/main.tsx:53]: move apiHost to env var

Comment (lenchq) [inline src/main.tsx:53]: move apiHost to env var

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "teleotp",
   "private": true,
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (lenchq) [inline package.json:4]: do a minor version bump instead (0.2.0)

**Review by lenchq — CHANGES_REQUESTED** [review]

**Review by lenchq — CHANGES_REQUESTED** [review]
need to also change version in package-lock.json

**Review by lenchq — APPROVED** [review]

**Comment by lenchq** [comment]
ready to merge

**Pull request merged**